root/etc/cont-init.d/45-plex-hw-transcode

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/with-contenv bash
+
+# Check for the existence of the Intel video device
+if [ -e /dev/dri ]; then
+	VIDEO_GID=$(stat -c '%g' /dev/dri/* | head -n 1)
+else
+	exit 0
+fi
+
+# Check if this GID matches the current plex user
+PLEX_GID=$(getent group plex | awk -F: '{print $3}')
+if [ "${PLEX_GID}" == "${VIDEO_GID}" ]; then
+	exit 0
+fi
+
+# Check if the GID is taken and swap to 65533
+CURRENT=$(getent group ${VIDEO_GID} | awk -F: '{print $1}')
+if [ -z "${CURRENT}" ] || [ "${CURRENT}" == 'video' ]; then
+	groupmod -g ${VIDEO_GID} video
```

Comment:
Could use a comment here describing something to the effect of `# Change/create the video group to match the GID of /dev/dri`